packages/vitest/src/integrations/chai/index.ts

```diff
@@ -15,7 +15,7 @@ import { getWorkerState } from '../../runtime/utils'
 import { createExpectPoll } from './poll'
 import './setup'
 
-export function createExpect(test?: TaskPopulated): ExpectStatic {
```

Comment:
This has to be considered a breaking change, right? The only place I could find where this is used with a parameter, it provides a `Test`, which extends `TaskPopulated`.

Could use `TaskPopulated | Test` instead and check if `test.type === 'test'` maybe.